features/checkout/paying_for_order/preventing_to_change_payment_method_of_already_cancelled_order.feature

```diff
@@ -0,0 +1,31 @@
+@paying_for_order
+Feature: Preventing to change payment method of the cancelled order
+    In order to not be able to change payment method of the order that is already cancelled
+    As a Customer
+    I want to be prevented from changing payment method of the cancelled order
```

Comment:
```suggestion
    In order to avoid invalid order data
    As a Shop Owner
    I want to prevent canceling an already canceled orders by customers
```

Some like this, maybe 🤔. Not being able to change a payment method sounds like a solution for me, not a goal.

Reply:
WDYT? 
```suggestion
    In order to avoid invalid order data
    As a Customer
    I should be prevented from changing payment method of the cancelled order
```

Reply:
As a customer I don't care whether Allegro or Amazon have invalid data. It's on their side to prevent me from doing stupid things. We might gather more opinions on this.

Reply:
hmmm, I more like the customer perspective, because the customer is the actor of this scenario (but ofc after all both sides benefit from this validation).
WDYT about sth like:
`
In order to perform only valid order operation
    As a Customer
    I should be prevented from changing payment method of the cancelled order
`